js/viewer/viewer.js

```diff
@@ -814,9 +814,14 @@ function Viewer(parent, config) {
 
     // Drag to draw event handler end
     this.onMouseDragToDrawEnd = function(event){
+        
+        if (event.userData.annotation._tag == "polygon") {
```

Comment:
Instead of checking _tag, check for userData.type (assuming they are the same thing.) 

In pure Javascript, you cannot define private properties. So we prepend attribute names with `_` to indicate that they are private and are not supposed to be used. In this case `_tag` is supposed to be a private property of `annotation`

Reply:
Yes, `userData.type` can provide the same information. So changed it to `userData.type`.